tests/unit/python/foglamp/services/core/api/test_support.py

```diff
@@ -14,7 +14,7 @@
 import pytest
 
 from foglamp.services.core import routes
-from foglamp.services.core.api import support
```

Comment:
This import should not be deleted, hence causes a `test_support` failure.
Fixed in https://github.com/foglamp/FogLAMP/pull/635